datadog/internal/utils/api_instances_helper.go

```diff
@@ -347,12 +347,12 @@ func (i *ApiInstances) GetAuthNMappingsApiV2() *datadogV2.AuthNMappingsApi {
 	return i.authNMappingsApiV2
 }
 
-// GetCloudWorkloadSecurityApiV2 get instance of CloudWorkloadSecurityApi
-func (i *ApiInstances) GetCloudWorkloadSecurityApiV2() *datadogV2.CloudWorkloadSecurityApi {
-	if i.cloudWorkloadSecurityApiV2 == nil {
-		i.cloudWorkloadSecurityApiV2 = datadogV2.NewCloudWorkloadSecurityApi(i.HttpClient)
+// GetCSMThreatsApiV2 get instance of CSMThreatsApi
+func (i *ApiInstances) GetCSMThreatsApiV2() *datadogV2.CSMThreatsApi {
```

Comment:
The underlying datadog package had a rename from datadogV2.CloudSecurityWorkloadApi to datadogV2.CSMThreatsApi so I did a find and replace to update the name